README.md

```diff
@@ -12,11 +12,23 @@ nvm. See https://github.com/creationix/nvm for more info.
 
 ## Get started
 
-Run:
-
 * npm install
 * npm run dev
 
+
+## NPM scripts
+
+| Script                 | Description                                      |
+|------------------------|--------------------------------------------------|
+| npm start              |  Spins up the express server                     |
+| npm test               |  Runs the tests                                  |
+| npm run build          |  Builds the lib                                  |
+| npm run dev            |  Spins up the dev server (Express + webpack)     |
```

Comment:
This is a nitpick, but "Start the dev server" is clearer and probably more ESL-friendly.